sdks/python/README.md

```diff
@@ -123,7 +123,7 @@ directory you created.
 
 ### Execute a pipeline locally
 
-The Apache Beam [examples](https://github.com/apache/beam/tree/python-sdk/sdks/python/apache_beam/examples) directory has many examples. All examples can be run locally by passing the arguments required by the example script.
+The Apache Beam [examples](https://github.com/apache/beam/tree/blob/master/sdks/sdks/python/apache_beam/examples) directory has many examples. All examples can be run locally by passing the arguments required by the example script.
```

Comment:
Thank you for catching this! Something went wrong with my replace I guess. Fixed this and verified the other links I updated.

PTAL.